docs/03-cli-reference/05-info.md

```diff
@@ -24,7 +24,8 @@ serverless info
 
 ### AWS
 
-On AWS the info plugin uses the `Outputs` section of the CloudFormation stack. Outputs will include Lambda function ARN's, a `ServiceEndpoint` for the API Gateway endpoint and user provided custom Outputs.
+On AWS the info plugin uses the `Outputs` section of the CloudFormation stack and the AWS SDK to gather the necessary information.
```

Comment:
Ah you're right, was confused there with the mention of removed Outputs that are added for the api key. GTM then